app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -1,10 +1,16 @@
-.gem-c-input {
+// scss-lint:disable QualifyingElement
+.gem-c-input,
+
+// Explicitly set the input type so that we have higher specificity than
+// https://github.com/alphagov/static/blob/79d29b6a4221874ead27e67d007704a6be166a57/app/assets/stylesheets/helpers/_buttons.scss#L91-L122
+// This can be removed once everthing is using `core_layout`.
+input[type=text].gem-c-input {
   @include core-19;
 
   box-sizing: border-box;
   width: 100%;
   height: 2.10526em;
-  margin-top: 0;
+  margin: 0;
```

Comment:
Can you put a comment on this margin change to indicate it's overriding like above?

```css
// Override unwanted global cascaded styles [1]
.selector {
...
margin-top: 0;
margin: 0; // [1]
```